SPECS-EXTENDED/kernel-rt/config

```diff
@@ -192,7 +193,7 @@ CONFIG_ARCH_SUPPORTS_NUMA_BALANCING=y
 CONFIG_ARCH_WANT_BATCHED_UNMAP_TLB_FLUSH=y
 CONFIG_CC_HAS_INT128=y
 CONFIG_CC_IMPLICIT_FALLTHROUGH="-Wimplicit-fallthrough=5"
-CONFIG_GCC11_NO_ARRAY_BOUNDS=y
+CONFIG_GCC10_NO_ARRAY_BOUNDS=y
```

Comment:
just making sure, is it intentional to go from gcc11 to gcc10?

Reply:
similar update in main kernel config in May upgrading from 6.6.22 to 6.6.29